packages/devtools_app/lib/src/debugger/flutter/debugger_service.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:vm_service/vm_service.dart';
+
+import '../../globals.dart';
+
+class DebuggerService {
```

Comment:
What purpose is the DebuggerService class serving? It seems to mainly just be forwarding messages from serviceManager.service.